File: sources/dialog_button_add.cc
cancel_button = dialog->add_button (
                                    Gtk::Stock::Cancel,
                                    Gtk::Response::Cancel);
 
ok_button = dialog->add_button (
                                Gtk::Stock::Ok,
                                Gtk::Response::Ok);
  
ok_button->grab_default ();

help_button = dialog->add_button (
                                  Gtk::Stock::Help,
                                  Gtk::Response::Help);

File: sources/dialog_button_add.py
cancel_button = dialog.add_button(

                Gtk.STOCK_CANCEL,
                Gtk.ResponseType.CANCEL);

ok_button = dialog.add_button(

            Gtk.STOCK_OK,
            Gtk.ResponseType.OK);

ok_button.grab_default();

help_button = dialog.add_button(

              Gtk.STOCK_HELP,
              Gtk.ResponseType.HELP);
